packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1562,21 +1570,74 @@ export default class Meeting extends StatelessWebexPlugin {
    * @returns {undefined}
    * @public
    * @memberof Meeting
+   * @deprecated after v1.89.3
    */
   closeRemoteStream() {
-    return Media.stopStream(this.mediaProperties.remoteStream).then(() => {
+    LoggerProxy.logger.warn('Meeting:index#closeRemoteStream --> [DEPRECATION WARNING]: closeRemoteStream has been deprecated after v1.89.3');
+    this.closeRemoteTracks();
+  }
+
+  /**
+   * Removes the remote tracks on the class instance and triggers an event
+   * to developers
+   * @returns {undefined}
+   * @memberof Meeting
+   */
```

Comment:
we dont need this function , A developer should not close any remote tracks on there side

Reply:
You're right but this function is private. Since we are deprecating the public one,  it's here for backwards compatibility. An effort to minimize any unforeseen issues with removing the public API that's already been exposed.